README.md

```diff
@@ -63,21 +64,24 @@ All runtimes share a common basic structure, but each additionally adds runtime-
 ├── Dockerfile
 ├── example
 │   ├── (runtime-specific)
-├── example.sh
+├── deploy.sh
 ├── launch.sh
 ├── README.md
 └── server.X
 ```
 
-- `example` directory includes example script using language of the specific runtime. File `docker-compose.yml` is configuration to easily run example code with `docker-compose up`.
-- `Dockerfile` file holds an image definition that prepares environment for the runtime to run on. These images are usually based on Linux Alpine or Linux Ubuntu.
-- `server.X` file is a runtime-specific HTTP server implementation. File extention is different for every programming language, for instance, Python is `server.py`.
-- `build.sh` script builds source code into executable script. This ca be either package installations, or build process of the runtime.
-- `launch.sh` script take executable script and prepare server that will execute it by running HTTP server on port `3000`
-- `example.sh` script is all-in-one solution that builds a script and launches HTTP server. This script is really handy for simple usage without executor.
-- `README.md` is runtime documentation.
+| Name               	| Description                                                                                                                                           	|
+|--------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| example/           	| Contains a sample function to demonstrate the usage of the runtime server                                                                             	|
+| docker-compose.yml 	| Configuration to easily run the example code with `docker-compose up`                                                                                 	|
+| Dockerfile         	| Instructions to build a runtime, install it's dependencies and setup the runtime server. These images are usually based on official alpine or ubuntu. 	|
+| server.X           	| A HTTP server implemented in the respective runtime's language. File extension depends on your runtime. For instance, Python is `server.py`           	|
+| build.sh           	| Script responsible for building user code. This can be package installations, or any specific build process the runtime requires.                     	|
+| launch.sh          	| Script to launch the HTTP server on port `3000`. Additionally, it also copies the user supplied code to a directory accessible to the server.         	|
```

Comment:
`launch` takes `.tar.gz` file as input, extracts it and prepares it to be run (for example in node, it copies node_modules to HTTP server script, because that is now Node loads dependencies).

The last step of `launch` is `npm start` which runs the `server.js`.